Makefile

```diff
@@ -257,18 +258,18 @@ ${ISTIO_OUT}/istioctl-win.exe: depend
 	STATIC=0 GOOS=windows bin/gobuild.sh $@ istio.io/istio/pkg/version ./pilot/cmd/istioctl
 
 MIXER_GO_BINS:=${ISTIO_OUT}/mixs ${ISTIO_OUT}/mixc
-$(MIXER_GO_BINS): depend
+$(MIXER_GO_BINS):
```

Comment:
makes sense - doubt someone will target this directly from the makefile.

Reply:
Someone might not target these directly, but they might target something else (e.g., a docker image) that cites these as a dependency.  Depend should be very lightweight after the 1st invocation, e.g.:

<pre>make depend
~/istio/build/go/out ~/istio/build/go/src/istio.io/istio
Protocols: dict file ftp ftps gopher http https imap imaps ldap ldaps pop3 pop3s rtmp rtsp scp sftp smb smbs smtp smtps telnet tftp 
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed
100 74.0M  100 74.0M    0     0  18075      0  1:11:36  1:11:36 --:--:-- 37.6M
~/istio/build/go/src/istio.io/istio

make depend
make: Nothing to be done for 'depend'.
</pre>